integration-tests/src/tests/client/snapshots/integration_tests__tests__client__process_blocks__runtime_gas_price__100Tgas_purchased_gas_price before GasPriceHostFn.snap

```diff
@@ -0,0 +1,33 @@
+---
+source: integration-tests/src/tests/client/process_blocks.rs
+expression: old_outcome
+---
+ExecutionOutcomeView {
+    logs: [],
+    receipt_ids: [
+        `11111111111111111111111111111111`,
+    ],
+    gas_burnt: 2428023045199,
+    tokens_burnt: 2428023045199000000000,
+    executor_id: AccountId(
+        "test0",
+    ),
+    status: Failure(ActionError(ActionError { index: Some(0), kind: FunctionCallError(ExecutionError("Link Error: Error while importing \"env\".\"purchased_gas_price\": unknown import. Expected Function(FunctionType { params: [I64], results: [] })")) })),
+    metadata: ExecutionMetadataView {
+        version: 1,
+        gas_profile: Some(
+            [
+                CostGasUsed {
+                    cost_category: "WASM_HOST_COST",
+                    cost: "CONTRACT_LOADING_BASE",
+                    gas_used: 35445963,
+                },
+                CostGasUsed {
+                    cost_category: "WASM_HOST_COST",
+                    cost: "CONTRACT_LOADING_BYTES",
+                    gas_used: 57655500,
+                },
+            ],
+        ),
+    },
+}
```

Comment:
If we expect to use such tests frequently, we probably should come up with more compact and readable representation for execution outcome.